docs/03-apis/api-v2/knora-iris.md

```diff
@@ -208,12 +206,14 @@ follows:
   - Standoff tag:
     `http://rdfh.ch/PROJECT_SHORTCODE/RESOURCE_UUID/values/VALUE_UUID/STANDOFF_UUID`
   - XML-to-standoff mapping:
-    `http://rdfh.ch/PROJECT_SHORTCODE/mappings/MAPPING_NAME`
+    `http://rdfh.ch/projects/PROJECT_SHORTCODE/mappings/MAPPING_NAME`
   - XML-to-standoff mapping element:
-    `http://rdfh.ch/PROJECT_SHORTCODE/mappings/MAPPING_NAME/elements/MAPPING_ELEMENT_UUID`
-  - Project: `http://rdfh.ch/projects/PROJECT_SHORTCODE` (or `http://rdfh.ch/projects/PROJECT_UUID`)
+    `http://rdfh.ch/projects/PROJECT_SHORTCODE/mappings/MAPPING_NAME/elements/MAPPING_ELEMENT_UUID`
+  - Project: `http://rdfh.ch/projects/PROJECT_UUID`
   - Group: `http://rdfh.ch/groups/PROJECT_SHORTCODE/GROUP_UUID`
   - Permission:
     `http://rdfh.ch/permissions/PROJECT_SHORTCODE/PERMISSION_UUID`
   - Lists: `http://rdfh.ch/lists/PROJECT_SHORTCODE/LIST_UUID`
   - User: `http://rdfh.ch/users/USER_UUID`
+
+  DSP-API supports `PROJECT_UUID` which are UUID version 4 or 5.
```

Comment:
How is it with the other UUIDs? I assume that also LIST_UUID, GROUP_UUID etc. only allow version 4 or 5.

Reply:
You're right, somehow the newest version of this file which states it for all UUIDs wasn't pushed to the repo. Will change that.